modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/KotlinServerCodegen.java

```diff
@@ -353,6 +369,7 @@ public static class Constants {
         public static final String USE_MUTINY_DESC = "Whether to use Mutiny (should not be used with useCoroutines). This option is currently supported only when using jaxrs-spec library.";
         public static final String OMIT_GRADLE_WRAPPER = "omitGradleWrapper";
         public static final String OMIT_GRADLE_WRAPPER_DESC = "Whether to omit Gradle wrapper for creating a sub project.";
+        public static final String KOTLIN_SERIALIZATION = "Whether Kotlin serialization should be used";
```

Comment:
looks like this is the description.

should the key be something like "enableKotlinSerialization" ?